engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +231,44 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float function1(float parameter){
+        double p = (double)parameter;
+        return (float) pow(p,0.6) * 0.78f ;
+    }
+
+    private float function2(float parameter){
+       return 0.15f + (0.64f * parameter) - (0.006f * parameter * parameter);
+    }
+
     @Command(shortDescription = "Sets the height of the player", runOnServer = true,
             requiredPermission = PermissionManager.CHEAT_PERMISSION)
     public String playerHeight(@Sender EntityRef client, @CommandParam("height") float amount) {
         try {
             ClientComponent clientComp = client.getComponent(ClientComponent.class);
             CharacterMovementComponent move = clientComp.character.getComponent(CharacterMovementComponent.class);
-            if (move != null) {
+            CharacterComponent charComp=clientComp.character.getComponent(CharacterComponent.class);
+            EntityRef player = client.getComponent(ClientComponent.class).character;
```

Comment:
This can be replaced by a reference to `clientComp` which is retrieved in the first line of this try-catch block.

```suggestion
            EntityRef player = clientComp.character;
```